### ISSUE TITLE
Which version of nextcloud is supported by this app

### ISSUE THREAD

**Issue by Qintiy** [privacyidea/privacyidea-owncloud-app#114]
i am trying to install this app on NextCloud 26.0.1 and in readme is written that it supports 12 and 13. I am just wondering if this app is out of date??

I get this error in Loggs
`[no app in context] Error: Error: Class "OCP\App" not found at <<closure>>

0. /var/www/nextcloud/apps/twofactor_privacyidea/appinfo/app.php line 25
   OCA\TwoFactor_privacyIDEA\AppInfo\Application->registerSettings()
1. /var/www/nextcloud/lib/private/legacy/OC_App.php line 316
   require_once("/var/www/nextcl ... p")
2. /var/www/nextcloud/lib/private/legacy/OC_App.php line 192
   OC_App::requireAppFile()
3. /var/www/nextcloud/lib/private/legacy/OC_App.php line 141
   OC_App::loadApp()
4. /var/www/nextcloud/lib/private/legacy/OC_User.php line 128
   OC_App::loadApps()
5. /var/www/nextcloud/lib/base.php line 758
   OC_User::setupBackends()
6. /var/www/nextcloud/lib/base.php line 1165
   OC::init()
7. /var/www/nextcloud/index.php line 34
   require_once("/var/www/nextcloud/lib/base.php")
`

and the file /var/www/nextcloud/apps/twofactor_privacyidea/appinfo/app.php has thic content:
`<?php

/**
 * @author Cornelius K      lbel <cornelius.koelbel@netknights.it>
 * @author Lukas Matusiewicz <lukas.matusiewicz@netknights.it>
 *
 * @license AGPL-3.0
 *
 * This code is a free software: You can redistribute it and/or modify
 * it under the terms of the GNU Affero General Public License, version 3,
 * as published by the Free Software Foundation.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 * GNU Affero General Public License for more details.
 *
 * You should have received a copy of the GNU Affero General Public License, version 3,
 * along with this program. If not, see <http://www.gnu.org/licenses/>
 */

use OCA\TwoFactor_privacyIDEA\AppInfo\Application;
$app = new Application();
// register the configuration settings templates
$app->registerSettings();`

**Comment by nilsbehlen** [comment]
Sir, this is an owncloud app ;)

From what i know, the internals on owncloud and nextcloud are not the same anymore (and that is what is probably causing the `Class not found` error), so this app does not work with nextcloud.

We are currently working on a nextcloud app, which should then be available from the nextcloud store. But i do not have an ETA yet.

**Comment by Qintiy** [comment]
Thanks for the information. For the nextcloud app release information will be probably on official privacyidea website or is there a news or relase website for privacyidea and apps?

**Comment by nilsbehlen** [comment]
yes, or when a repo is added to the github org you could try it earlier, since i guess you know what you are doing, you dont need it to be in the store.

**Comment by Qintiy** [comment]
so for now to make it work on nextcloud the only solution is simplesamlphp am I right?

**Comment by nilsbehlen** [comment]
or keycloak, which is way better than ssp.
